python/ql/src/semmle/python/web/twisted/Twisted.qll

```diff
@@ -24,29 +21,32 @@ predicate isKnownRequestHandlerMethodName(string name) {
     name.matches("render_%")
 }
 
-/** Holds if `node` is likely to refer to an instance of the twisted
+/**
+ * Holds if `node` is likely to refer to an instance of the twisted
  * `Request` class.
  */
 predicate isTwistedRequestInstance(NameNode node) {
     node.pointsTo().getClass() = theTwistedHttpRequestClass()
     or
-    /* In points-to analysis cannot infer that a given object is an instance of
-     * the `twisted.web.http.Request` class, we also include any parameter
-     * called `request` that appears inside a subclass of a request handler
-     * class, and the appropriate arguments of known request handler methods.
-     */
+    /*
+      * In points-to analysis cannot infer that a given object is an instance of
```

Comment:
This misalignment of the asterisks puzzles me a bit. What version of the auto-format wrapper script are you using?

Reply:
not the right one :confused: I fixed my script, and fixed the commit :+1: 

Thanks for your ever watchful eye